targets/unity-v2/source/Shared/Internal/PlayFabErrors.cs.ejs

```diff
@@ -41,12 +41,25 @@ namespace PlayFab
 
         [ThreadStatic]
         private static StringBuilder _tempSb;
+         /// <summary>
+        /// This converts the PlayFabError into a human readable string describing the error.
+        /// If error is not found, it will return the http code, status, and error
+        /// </summary>
+        /// <returns>A description of the error that we just incur.</returns>
         public string GenerateErrorReport()
         {
             if (_tempSb == null)
                 _tempSb = new StringBuilder();
             _tempSb.Length = 0;
-            _tempSb.Append(ApiEndpoint).Append(": ").Append(ErrorMessage);
+            if (String.IsNullOrEmpty(ErrorMessage))
+            {
+                _tempSb.Append("Http Code: ").Append(HttpCode.ToString()).Append("\nHttp Status: ").Append(HttpStatus).Append("\nError: ").Append(Error.ToString()).Append("\n");
```

Comment:
This line should also likely include ApiEndpoint.
ApiEndpoint is the destination URL, and is also present if the result is (404, etc)